### PR TITLE
Fix visibility of WooCommerce checkboxes and radios

### PR DESCRIPTION
**Pull request by EvanHerman**
image

image

### CODE REVIEW

**Comment by richtabor** [comment]
I think you forgot to pull in master's changes (PostCSS to SCSS). We don't need the `&` preceded anymore.

**Review by AnthonyLedesma — APPROVED** [review]
Approved once the merge conflicts are resolved.

**Pull request merged**
